packages-next/keystone/src/lib/createSystem.ts

```diff
@@ -73,7 +73,7 @@ export function createKeystone(config: KeystoneConfig) {
   return keystone;
 }
 
-export function createSystem(config: KeystoneConfig): KeystoneSystem {
+export function createSystem(config: KeystoneConfig) {
```

Comment:
Should this be replaced with a substitute return type, instead of having the return type removed entirely?

Reply:
The return type is correctly inferred from the rest of the type system, so there's no need for an explicit return type.